src/attr.rs

```diff
@@ -66,9 +68,12 @@ pub(crate) fn parse_our_attr(attr: &Attribute) -> Result<OurAttr, ()> {
     let body = match &*tokens {
         [TokenTree::Group(g)] => g.stream(),
         _ => {
-            return attr.tokens.span()
-                .err(format!("expected single group delimitted by`()`, found '{:?}'", tokens))
-                .emit_with_attr_note();
+            emit_error!(
+                attr.tokens.span(),
+                "expected single group delimited by`()`, found '{:?}'",
```

Comment:
Add a space after `by`. Not your fault, but you could fix it ^_^

Reply:
Yep